tests/mpesa-rust/transaction_reversal_test.rs

```diff
@@ -0,0 +1,19 @@
+use crate::get_mpesa_client;
+
+#[tokio::test]
+async fn stk_push_test() {
+    let client = get_mpesa_client!();
+
+    let response = client
+        .transaction_reversal("testapi496")
+        .result_url("https://testdomain.com/ok")
+        .timeout_url("https://testdomain.com/err")
+        .transaction_id("OEI2AK4Q16")
+        .amount(1.0)
+        .receiver_party("600983")
+        .remarks("wrong recipient")
+        .send()
+        .await;
+
+    assert!(response.is_ok())
```

Comment:
Quick favour: could you enable ci jobs to run on `pull_request` too in `.github/workflows/general.yml` as part of your PR?

Reply:
wait just checked that `pull_request` is included in the list, will investigate why ci jobs are not running

Reply:
need to approve the PR before the jobs run, will change this.